_data/projects/quicknote.yml

```
@@ -0,0 +1,9 @@
+name: Quick Note
+desc: Quick Note is an open-source project for saving snippets of information to 'memory' files from the command line
+site: https://github.com/scamacho59/quicknote
```

:art: point to renamed repository?

```suggestion
site: https://github.com/scamacho23/quicknote
```